infrastructure/scripts/test_gpt5_api_access.py

```
@@ -0,0 +1,309 @@
+            end_time = time.time()
+            latency = (end_time - start_time) * 1000
+
+            result["available"] = True
+            result["response"] = (
+                response.get("choices", [{}])[0].get("message", {}).get("content", "")
+            )
+            result["latency_ms"] = round(latency, 2)
+
+            logger.info("model_test_success", model=model_name, latency_ms=result["latency_ms"])
+
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)